docs/guides/airnode/post-processing/index.md

```diff
@@ -0,0 +1,256 @@
+  }
+]
+```
+
+Below is a more human readable rendering of the post processing code. The
+`input` variable contains the response from the API provider's operation. The
+`output` variable must be set before the code exists as Airnode uses it to
+encode any values for use on-chain. See more about
+[Input and Output](/reference/ois/latest/processing.md#input-and-output) in the
+OIS reference.
+
+```js
+let avg_price = 0;
+let avg_market_cap = 0;
+input.prices.forEach((el) => {
+  avg_price = avg_price + el[1];
+});
+input.avg_price = (avg_price / 7).toString();
+input.market_caps.forEach((el) => {
+  avg_market_cap = avg_market_cap + el[1];
+});
+input.avg_market_cap = (avg_market_cap / 7).toString();
+output = input;
+```
+
+## 4. Remove the existing Docker container
+
+The Docker container you created in step #1 must be deleted and a new one
+created which uses the newer `config.jon` file. From within the Docker desktop
+app find the docker container named **quick-start-container-airnode** and delete
+it.
+
+<img src="./src/delete-container.png"/>
+
+## 5. Redeploy the Docker container
+
+From your project root run the Docker container deploy command. It will use the
+newer `config.json` file you have created.
+
+::: code-group
+
+```sh [Mac/WSL2/PowerShell]
+docker run \
+  --volume "$(pwd):/app/config" \
+  --name quick-start-container-airnode \
+  --publish 3000:3000 \
+  api3/airnode-client:latest
+```
+
+```batch [Windows CMD]
+docker run ^
+  --volume "%cd%:/app/config" ^
+  --name quick-start-container-airnode ^
+  --publish 3000:3000 ^
+  api3/airnode-client:latest
+```
+
+```sh [Linux (host networking)]
+docker run \
+  --volume "$(pwd):/app/config" \
+  --name quick-start-container-airnode \
+  --network host \
+  api3/airnode-client:latest
+```
+
+:::
+
+Leave the command active in the terminal window. It will display the log items
+from the container. If you wish to close it then use the log monitor from within
+the Docker desktop app by selecting **View details** from the container's
+**Action** menu.
+
+<img src="./src/view-logs.png" style="width:70%;border:solid 1px lightgray;"/>
+
+```sh
+% docker run \
+  --volume "$(pwd):/app/config" \
+  --name quick-start-container-airnode \
+  --publish 3000:3000 \
+  api3/airnode-client:latest
+WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
+[2023-06-06 21:49:44.565] INFO Gateway "httpSignedDataGateway" not enabled.
+[2023-06-06 21:49:44.586] INFO Gateway "oevGateway" not enabled.
+[2023-06-06 21:49:44.596] INFO HTTP (testing) gateway listening for request on "http://localhost:3000/http-data/01234567-abcd-abcd-abcd-012345678abc/:endpointId"
+[2023-06-06 21:49:44.607] INFO API gateway server running on "http://localhost:3000"
+
```

Comment:
I would suggest limiting this code block to only the relevant line, which is the one with the gateway URL, and changing the text to highlight that the user needs this URL for the next step.